iceoryx_posh/test/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 cmake_minimum_required(VERSION 3.5)
-project(test_iceoryx_posh VERSION ${test_iceoryx_posh})
+set(test_iceoryx_posh_VERSION 0.16.1)
```

Comment:
We should not set version numbers for test libs because its harder to maintain and we have to set more version numbers in different CMake files if a release come.
The tests are always attached to a specific release version, therefore i would not set it. Additionally they are already versioned with git tags.

Reply:
@dkroenke I will set the version to zero